### DISCUSSION TITLE
No Error Handling for Uploading document other then JPEG and PNG format

### DISCUSSION THREAD

**Discussion by AKACHI-4** [community/community#110459] [GitHub Education]
### Hey Github Team,

I was re-verifying or renewing my GitHub Student Developer Pack, and I got stuck. I was uploading the document, and after clicking to process my application, it kept me on the same page without any message indicating what was going wrong. Man, I tried uploading and submitting the application 10-15 times, but I encountered the same behavior repeatedly.

*However, I found that the problem lies with the file format. I was uploading JPG, but it required JPEG or PNG. There should be a message or popup indicating that the file should be in the given format only, and that I was uploading the wrong format.*

**This feature should be there so that at least the user can have a better experience overall.**

**Comment by luq7** [comment]
How come you could upload a document? While I was only provided the option to take a photo of some docs? I think that the design is very counter intuitive. Should have a place for us to request an update

**Reply by AKACHI-4** [reply]
> How come you could upload a document? While I was only provided the option to take a photo of some docs?

Sry, I mean verification document, For obvious format is jpeg/png

Yep .. not sure what could be the right raise for design chores but hopefully someone from team will look into it.
